tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -808,14 +812,15 @@ INSTANTIATE_TEST_SUITE_P(
             /*yuv_depth=*/Values(8, 10),
             /*width=*/Values(1, 2, 127, 200),
             /*height=*/Values(1, 2, 127, 200),
-            Values(AVIF_RGB_FORMAT_RGB, AVIF_RGB_FORMAT_ARGB),
+            Values(AVIF_RGB_FORMAT_RGB, AVIF_RGB_FORMAT_RGBA),
             Range(AVIF_PIXEL_FORMAT_YUV444, AVIF_PIXEL_FORMAT_COUNT),
             // Test an odd and even number for threads. Not adding all possible
             // thread values to keep the number of test instances low.
             /*threads=*/Values(2, 7),
             /*avoidLibYUV=*/Values(true, false),
             Values(AVIF_CHROMA_UPSAMPLING_FASTEST,
-                   AVIF_CHROMA_UPSAMPLING_BILINEAR)));
+                   AVIF_CHROMA_UPSAMPLING_BILINEAR),
+            /*has_alpha*/ Values(true, false)));
```

Comment:
Optional: In libaom I have seen `::testing::Bool()` used, which is equivalent to `Values(false, true)`. You can use `Values(true, false)` to match existing usage (such as line 820).

Reply:
Done. Replaced the new usage with ::testing::Bool().

There are a few other instances of Values(true, false). I will change them all to ::testing::Bool() in a separate PR.